src/styles/block/card.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+}
+
+.container__card {
+  padding: 0 16px;
+}
```

Comment:
All this styles you can use in **.card** They determine the internal positioning of the elements in the block, not the external position of block on the page.